src/java.base/share/classes/java/security/SignatureSpi.java

```diff
@@ -381,32 +383,34 @@ protected abstract void engineSetParameter(String param, Object value)
      *
      * @throws    InvalidAlgorithmParameterException if this method is
      * overridden by a provider and the given parameters
-     * are inappropriate for this signature engine
+     * are inappropriate for this signature object
      */
     protected void engineSetParameter(AlgorithmParameterSpec params)
             throws InvalidAlgorithmParameterException {
         throw new UnsupportedOperationException();
     }
 
     /**
+     * Returns the parameters used with this signature object.
+     *
      * <p>This method is overridden by providers to return the parameters
-     * used with this signature engine.
+     * used with this signature object.
```

Comment:
The 2 paragraphs above look the same. Of course, if you believe the 1st paragraph should always be a simple description, that's OK.

Reply:
Yes, I will remove the second sentence to match the javadoc of other methods.